src/Sylius/Behat/Element/Shop/CheckoutSubtotalElement.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Element\Shop;
+
+use FriendsOfBehat\PageObjectExtension\Element\Element;
+
+final class CheckoutSubtotalElement extends Element implements CheckoutSubtotalElementInterface
```

Comment:
```suggestion
final class CheckoutSummaryElement extends Element implements CheckoutSummaryElementInterface
```
?

Reply:
The whole element in HTML is named `checkout-subtotal`, so I've picked this name.